gallery/gallery-library/src/main/java/org/silverpeas/components/gallery/MediaUtil.java

```diff
@@ -212,138 +207,117 @@ public static void processPhoto(final FileHandler fileHandler, final Photo photo
    * @param watermarkOther the secondary metadata retrieved to compute the watermark
    * @throws Exception
    */
-  public static void processPhoto(final FileHandler fileHandler, final Photo photo,
+  public synchronized static void processPhoto(final FileHandler fileHandler, final Photo photo,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AModifiersOrderCheck)